core/openshift-pipelines/kustomization.yaml

```diff
@@ -3,4 +3,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 bases:
-  - github.com/redhat-cop/gitops-catalog/openshift-pipelines-operator/overlays/stable?ref=main
+  - github.com/redhat-cop/gitops-catalog/openshift-pipelines-operator/overlays/pipelines-1.8?ref=main
```

Comment:
Any objections to using "latest", and use pipelines-<version> with a patch override?  That way we don't have to update this here when 1.8 is eol.  Thoughts?

Reply:
Was saying let's use latest versus a specific version which is likely to become outdated eventually.  Sounds like we are saying the same thing :)